nucliadb/nucliadb/search/search/query.py

```diff
@@ -51,9 +51,47 @@
 from nucliadb_utils import const
 from nucliadb_utils.utilities import has_feature
 
+from .exceptions import InvalidQueryError
+
 ENTITY_FILTER_PREFIX = "/e/"
 LABEL_FILTER_PREFIX = "/l/"
 
+LABEL_QUERY_ALIASES = {
+    # aliases to make querying labels easier
+    "tags": ["t"],
+    "labels": ["l"],
```

Comment:
Thought: should we use `classification-labels` instead? To be more specific.

Reply:
or even just "classification" I guess right?